scripts/mkcloud

```diff
@@ -540,12 +540,36 @@ function setupadmin()
     fi
 }
 
+function wait_for_node_shutdown()
+{
+    wait_for 150 1 "virsh domstate $1 | grep shut.off" '$1 node to shut down'
+}
+
+function remove_snapshot_volume()
+{
+    if [[ "${1}" =~ \.snap$ ]]; then
```

Comment:
yes, that might make more sense

and btw. `if [[ $1 =~ \.snap$ ]]` works without quotes